docs/docs/contributing/howtos.mdx

```diff
@@ -538,18 +538,17 @@ pybabel init -i superset/translations/messages.pot -d superset/translations -l f
 
 ### Extracting new strings for translation
 
-This step needs to be done every time application strings change. This happens fairly
-frequently, so if you want to ensure that your translation has good coverage, this
-step needs to be run fairly frequently and the updated strings merged to the upstream
-codebase via PRs. To update the template file `superset/translations/messages.pot`
-with current application strings, run the following command:
+Periodically, when working on translations, we need to extract the strings from both the
+backend and the frontend to compile a list of all strings to be translated. It doesn't
+happen automatically and is a required step to gather the strings and get them into the
+`.po` files where they can be translated, so that they can then be compiled.
+
+This script does just that:
 
 ```bash
-pybabel extract -F superset/translations/babel.cfg -o superset/translations/messages.pot -k _ -k __ -k t -k tn -k tct .
+./scripts/translations/babel_update.sh
```

Comment:
Just capturing a note for one of us to open a PR later. It seems that if we touch the .pot file, the .po and .json/.mo files have a huge diff, as things aren't ordered deterministically. It might be good to add a sorting step in the file output, via this script or some other place, e.g.:

JSON:
 `jq -S . input.json > output.json`
 
 PO:
 ```python
 import polib

po = polib.pofile('path_to_file.po')
po.save_as_mofile('path_to_file.mo')  # Save if needed
sorted_entries = sorted(po, key=lambda x: x.msgid)
po[:] = sorted_entries
po.save('path_to_file.po')
 ```